models/issue_assignees.go

```diff
@@ -192,7 +182,7 @@ func (issue *Issue) changeAssignee(sess *xorm.Session, doer *User, assigneeID in
 		}
 		if err := prepareWebhooks(sess, issue.Repo, HookEventPullRequest, apiPullRequest); err != nil {
```

Comment:
The webhooks part doesn't seem related with changing assignees. I thihk it should be moved to another function.

Reply:
Actually it is, below is help text for PR trigger event. :wink:

> Pull request opened, closed, reopened, edited, approved, rejected, review comment, **assigned, unassigned**, label updated, label cleared or synchronized.

Reply:
> 
> 
> Actually it is, below is help text for PR trigger event. 😉
> 
> > Pull request opened, closed, reopened, edited, approved, rejected, review comment, **assigned, unassigned**, label updated, label cleared or synchronized.

Oh, that's not what I meant. 😄 What I meant is that this code seems to belong to `services` rather than `models`. Anyway, it was here before, so refactoring should be for another PR.

Reply:
Aha, yes you are probably right, but I will leave out that refactoring from this PR. I think we should have some design guidelines what type of code should be where. I think I've started to understand Gitea's code structure now....